src/api/auth.js

```diff
@@ -25,6 +25,7 @@ const nativeH = require('../native/helpers');
 const types = require('../native/types');
 const h = require('../helpers');
 const inTesting = require('../consts').inTesting;
+const { SignKey, EncKey } = require('./misc');
```

Comment:
Exactly, just so I can return a SignKey ref from both Auth and MutableData functions.